### PR TITLE
DOCS: PostGIS docs - add new install script information

### PR DESCRIPTION
**Pull request by mkiyama**
PostGIS topic has been updated to reflect the following
--PostGIS extension includes a new install/uninstall script.
--PostGIS SQL scripts to install/uninstall PostGIS and PostGIS raster have been moved.

### CODE REVIEW

File: gpdb-doc/dita/ref_guide/extensions/postGIS.xml [hunk 1]
```diff
@@ -94,97 +98,55 @@
...
+            <codeph>uninstall</codeph> option to remove PostGIS and PostGIS Raster. This example
+          removes PostGIS and PostGIS Raster support from the database
+          <codeph>mydatabase</codeph>.<codeblock><codeph>postgis_manager.sh</codeph> mydatabase uninstall</codeblock></p>
+        <p>The script runs both the PostGIS SQL scripts that remove PostGIS and PostGIS Raster from
+          a database: <codeph>uninstall_rtpostgis.sql</codeph> and
+            <codeph>uninstall_postgis.sql</codeph>.</p>
+        <p>The <codeph>postgis_manager.sh</codeph> script does not remove these PostGIS Raster
+          environment variables the <codeph>greenplum_path.sh</codeph> file:
+            <codeph>GDAL_DATA</codeph>, <codeph>POSTGIS_ENABLE_OUTDB_RASTERS</codeph>,
+            <codeph>POSTGIS_GDAL_ENABLED_DRIVERS</codeph>. The environment variables are removed
```
Comment (mkiyama) [inline gpdb-doc/dita/ref_guide/extensions/postGIS.xml]: I just tried it with the package from PivNet.
Uninstalling the PostGIS extension with the gppkg utility seems to remove the  GDDAL environment variables.
Am I missing something?

Comment (orhankislal) [inline gpdb-doc/dita/ref_guide/extensions/postGIS.xml]: @mkiyama I double-checked and confirmed that this is correct, `gppkg -r` does remove the lines. Sorry, I thought this was supposed to be handled by `postgis_manager.sh`

Comment (mkiyama) [inline gpdb-doc/dita/ref_guide/extensions/postGIS.xml]: @orhankislal 
There are build instructions for building PostGIS from the geospatial repo.
Will they work, are they current?

Comment (orhankislal) [inline gpdb-doc/dita/ref_guide/extensions/postGIS.xml]: I am updating the README on the geospatial repo right now. 
LGTM

File: gpdb-doc/dita/ref_guide/extensions/postGIS.xml [hunk 2]
```diff
@@ -94,97 +98,55 @@
...
-  $GPHOME/share/postgresql/contrib/postgis-2.1/rtpostgis.sql
-psql -d mydatabase -f 
-  $GPHOME/share/postgresql/contrib/postgis-2.1/spatial_ref_sys.sql</codeblock>
-      <p>After running the scripts, the database is spatially enabled and is PostGIS Raster
-        enabled.</p>
+      <p>The Greenplum Database PostGIS extension contains the <codeph>postgis_manager.sh</codeph>
+        script that installs or removes both the PostGIS and PostGIS Raster features in a database.
+        After the PostGIS extension package is installed, the script is in
+          <codeph>$GPHOME/share/postgresql/contrib/postgis-2.1/</codeph>. The
+          <codeph>postgis_manager.sh</codeph> script runs SQL scripts that install or removes
```
Comment (dyozie) [inline gpdb-doc/dita/ref_guide/extensions/postGIS.xml]: removes -> remove

File: gpdb-doc/dita/ref_guide/extensions/postGIS.xml [hunk 3]
```diff
@@ -336,4 +298,112 @@ SELECT AddGeometryColumn('public', 'geometries', 'geom', 0, 'LINESTRING', 2);</c
...
+      </topic>
+    </topic>
+    <topic id="topic_xp2_lgl_r1b">
+      <title>Scripts that Remove PostGIS and PostGIS Raster Support</title>
+      <body>
+        <p>To remove PostGIS support from a database, run SQL scripts that are supplied with the
+          PostGIS extension package in
+            <codeph>$GPHOME/share/postgresql/contrib/postgis-2.1/</codeph>
+        </p>
+        <note>If you installed PostGIS Raster, you must uninstall the PostGIS Raster before you
```
Comment (dyozie) [inline gpdb-doc/dita/ref_guide/extensions/postGIS.xml]: "the PostGIS Raster" -> "PostGIS Raster"

File: gpdb-doc/dita/ref_guide/extensions/postGIS.xml [hunk 4]
```diff
@@ -336,4 +298,112 @@ SELECT AddGeometryColumn('public', 'geometries', 'geom', 0, 'LINESTRING', 2);</c
...
+        <p>These scripts remove PostGIS and PostGIS Raster objects from a database.<ul
+            id="ul_yp2_lgl_r1b">
+            <li><codeph>uninstall/uninstall_rtpostgis.sql</codeph> - Removes the PostGIS Raster
+              object and function definitions.</li>
+            <li><codeph>uninstall/uninstall_postgis.sql</codeph> - Removes the PostGIS objects and
+              function definitions.</li>
+          </ul></p>
+        <p>After PostGIS support has been removed from all databases in the Greenplum Database
+          system, you can remove the PostGIS extension package. For example this
+            <codeph>gppkg</codeph> command removes PostGIS extension package.
```
Comment (dyozie) [inline gpdb-doc/dita/ref_guide/extensions/postGIS.xml]: "removes **the** PostGIS extension package/"

File: gpdb-doc/dita/ref_guide/extensions/postGIS.xml [hunk 5]
```diff
@@ -51,10 +55,10 @@
       <title id="ij168742">Greenplum PostGIS Extension</title>
       <body>
         <p>The Greenplum Database PostGIS extension package is available from <xref
```
Comment (dyozie) [inline gpdb-doc/dita/ref_guide/extensions/postGIS.xml]: Conditionalize this as otherprops="pivotal"

Comment (mkiyama) [inline gpdb-doc/dita/ref_guide/extensions/postGIS.xml]: @dyozie 
Did you want the whole paragraph or just the first sentence conditionalized?

Comment (dyozie) [inline gpdb-doc/dita/ref_guide/extensions/postGIS.xml]: I just meant the phrase regarding pivnet downloads.  Thanks!

**Comment by mkiyama** [comment]
@dyozie - updated based on review comments.

**Pull request merged**